source/generate.py

```diff
@@ -271,33 +255,34 @@ def check_if_prompt_inst_in_tokenizer_vocab(tokenizer, inst_prompt_list):
     model_repo = "misnaej/the-jam-machine-elec-famil"
     generated_sequence_files_path = define_generation_dir(model_repo)
 
-    TEMPERATURE = 0.75
-    INSTRUMENT_PROMPT_LIST = [81, 28, "32", "DRUMS"]
-    DENSITY_LIST = [3, 2, 3, 3]
+    temperature = 0.2
+    instrument_promt_list = ["0", "4", "DRUMS"]
+    # instrument_promt_list = ["32", "30", "DRUMS"]
+    density_list = [3, 2, 3]
 
     """" load model and tokenizer """
```

Comment:
This is the wrong type of comment in this context. There are two types of comments in python:
- regular comments start with # and describe code logic outside of definitions
- triple quotes ( """ ) are called docstrings and used exclusively under the definition of a class, function or at the top of the script to describe the whole file